### DISCUSSION TITLE
How to get files from the config dir into the containers?

### DISCUSSION THREAD

**Discussion by gerricom** [postalserver/postal#2032] [Help with using Postal]
Hello everybody!

I'm trying to make our Postal SMTP server talk TLS and therefore I copied a key and certificate to `/opt/postal/config`. I doublechecked permissions but those files won't show up in the containers. If I `postal bash smtp` and then change into the config dir, it still stays empty. Yes, I restarted postal... 

What am I doing wrong here? And why is the config dir fully empty in the container? Shouldn't I at least find the postal.yml there?

**Comment by willpower232** [comment]
As per the install [docker-compose.yml](https://github.com/postalserver/install/blob/ce3ddfa66e18a7509ebec9c7aebf84fcd70675df/templates/docker-compose.yml#L8) the folder _inside_ the container is `/config`

**Reply by gerricom** [reply]
Yes, so far I understand this. But the folder inside the container stays empty... I'm not very used to docker but is there any command to force mount my directories? Or are there any logs where I can see why the config directory is not mounted?

**Reply by willpower232** [reply]
if `postal dc app ls /config` is definitely empty and `ls /opt/postal/config` definitely isn't then you'll need something like `postal stop` and `postal start` to reconnect it

**Reply by gerricom** [reply]
`postal dc app ls /config` this command seems to be not correct... 🤔

**Reply by willpower232** [reply]
oh I can't read, `postal dc exec app ls /config`

**Reply by gerricom** [reply]
Ah this works now. Thanks! But the dir is still empty... even after stop/start... 😔

**Reply by willpower232** [reply]
I guess the problem could be related to docker volumes being screwy? You might have to dig around in docker and remove them to let it recreate them properly (don't delete any database volume though :grimacing: ).

**Reply by gerricom** [reply]
Hmmmm... I really don't know, what happened, but fiddling around today suddenly I got the files I wish inside the container. Yay! Thanks for your patience ;)
